content/scalability/docs/workloads.md

```diff
@@ -97,3 +97,29 @@ request:
 Mutating webhooks can modify resources in frequent succession. If you have 5 mutating webhooks and deploy 50 resources etcd will store all versions of each resource until compaction runs—every 5 minutes—to remove old versions of modified resources. In this scenario when etcd removes superseded resources there will be 200 resource version removed from etcd and depending on the size of the resources may use considerable space on the etcd host until defragmentation runs every 15 minutes.
 
 This defragmentation may cause pauses in etcd which could have other affects on the Kubernetes API and controllers. You should avoid frequent modification of large resources or modifying hundreds of resources in quick succession.
+
+## Compare the workloads across 2 clusters
+
+If you have 2 clusters that you think they should have similar performance, but they don't. You should compare the metrics to
+figure out the difference.
+
+A very common question is why the clusters have different latency. It is usually caused by the different volumes of API
+requests. You can run the following CloudWatch LogInsight query to understand the difference.
```

Comment:
Nit:
> It is usually caused by the different volumes of API
requests.

Wondering if you want to phrase it this way ? : " It is usually caused by difference in the volume of API requests."

Reply:
Updated